wallet/AndroidManifest.xml

```diff
@@ -1,244 +1,257 @@
-			android:theme="@style/My.Theme.ChildActivity" />
-		<activity
-			android:name="de.schildbach.wallet.ui.RestoreWalletActivity"
-			android:configChanges="orientation|keyboard|keyboardHidden"
-			android:theme="@style/My.Theme.Dialog" >
-			<intent-filter android:label="@string/import_keys_intent_filter_action" >
-				<action android:name="android.intent.action.VIEW" />
-
-				<category android:name="android.intent.category.DEFAULT" />
-
-				<data android:mimeType="application/x-darkcoin-wallet-backup" />
-			</intent-filter>
-		</activity>
-		<activity
-			android:name="de.schildbach.wallet.ui.ScanActivity"
-			android:configChanges="orientation|keyboard|keyboardHidden"
-			android:screenOrientation="landscape"
-			android:stateNotNeeded="true"
-			android:theme="@style/My.Theme.Fullscreen"
-			android:windowSoftInputMode="stateAlwaysHidden" />
-		<activity
-			android:name="de.schildbach.wallet.ui.send.SweepWalletActivity"
-			android:configChanges="keyboard|keyboardHidden"
-			android:label="@string/sweep_wallet_activity_title"
-			android:screenOrientation="behind"
-			android:theme="@style/My.Theme.ChildActivity"
-			android:windowSoftInputMode="adjustResize" />
-
-		<provider
-			android:name="de.schildbach.wallet.AddressBookProvider"
-			android:authorities="${applicationId}.address_book"
-			android:exported="false" />
-		<provider
-			android:name="de.schildbach.wallet.ExchangeRatesProvider"
-			android:authorities="${applicationId}.exchange_rates"
-			android:exported="false" />
-		<provider
-			android:name="de.schildbach.wallet.FileAttachmentProvider"
-			android:authorities="${applicationId}.file_attachment"
-			android:exported="false"
-			android:grantUriPermissions="true" />
-
-		<receiver
-			android:name="de.schildbach.wallet.WalletBalanceWidgetProvider"
-			android:label="Dash Wallet Balance" >
-			<intent-filter>
-				<action android:name="android.appwidget.action.APPWIDGET_UPDATE" />
-			</intent-filter>
-
-			<meta-data
-				android:name="android.appwidget.provider"
-				android:resource="@xml/wallet_balance_widget" />
-		</receiver>
-
-		<service
-			android:name="de.schildbach.wallet.service.BlockchainServiceImpl"
-			android:exported="false" />
-		<service
-			android:name="de.schildbach.wallet.offline.AcceptBluetoothService"
-			android:exported="false" />
-
-		<receiver android:name="de.schildbach.wallet.service.BootstrapReceiver" >
-			<intent-filter>
-				<action android:name="android.intent.action.BOOT_COMPLETED" />
-			</intent-filter>
-			<intent-filter>
-				<action android:name="android.intent.action.MY_PACKAGE_REPLACED" />
-			</intent-filter>
-		</receiver>
-
-		<service
-			android:name="de.schildbach.wallet.service.UpgradeWalletService"
-			android:exported="false" />
-
-		<service
-			android:name="de.schildbach.wallet.service.InactivityNotificationService"
-			android:exported="false" />
-
-		<meta-data
-			android:name="android.nfc.disable_beam_default"
-			android:value="true" />
-	</application>
```

Comment:
Don’t make white space changes to this file.  We need to keep it the same as Bitcoin Wallet.